### PR TITLE
Fix dashboard metrics labels

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: grafana-dashboard/overlays/moc-prod/thoth-sli-slo.json [hunk 1]
```diff
@@ -94,7 +94,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "up{field=\"user-api-thoth-frontend-prod.apps.smaug.na.operate-first.cloud\", prometheus=\"moc/smaug\"}",
+          "expr": "up{field=\"user-api-thoth-frontend-prod.apps.apps.smaug.na.operate-first.cloud\", cluster=\"$env\"}",
```
Comment (harshad16) [inline grafana-dashboard/overlays/moc-prod/thoth-sli-slo.json:97]:
```suggestion
          "expr": "up{field=\"user-api-thoth-frontend-prod.apps.smaug.na.operate-first.cloud\", cluster=\"$env\"}",
```

File: grafana-dashboard/overlays/moc-prod/thoth-sli-slo.json [hunk 2]
```diff
@@ -183,7 +183,7 @@
...
+          "expr": "up{field=\"user-api-thoth-frontend-prod.apps.apps.smaug.na.operate-first.cloud\", cluster=\"$env\"} == 1 and on(cluster) up{field=\"investigator-faust-web-thoth-infra-prod.apps.smaug.na.operate-first.cloud\", cluster=\"$env\"} == 1  and on(prometheus) up{field=\"workflow-controller-metrics-thoth-backend-prod.apps.smaug.na.operate-first.cloud\", cluster=\"$env\"} and on(cluster) thoth_openshift_connection_issues{cluster=\"$env\"} and on(cluster) thoth_graphdb_connection_issues{cluster=\"$env\"} and on(cluster) thoth_kafka_connection_issues{cluster=\"$env\"} and on(cluster) thoth_ceph_connection_issues{cluster=\"$env\"}",
```
Comment (harshad16) [inline grafana-dashboard/overlays/moc-prod/thoth-sli-slo.json:186]:
```suggestion
          "expr": "up{field=\"user-api-thoth-frontend-prod.apps.smaug.na.operate-first.cloud\", cluster=\"$env\"} == 1 and on(cluster) up{field=\"investigator-faust-web-thoth-infra-prod.apps.smaug.na.operate-first.cloud\", cluster=\"$env\"} == 1  and on(prometheus) up{field=\"workflow-controller-metrics-thoth-backend-prod.apps.smaug.na.operate-first.cloud\", cluster=\"$env\"} and on(cluster) thoth_openshift_connection_issues{cluster=\"$env\"} and on(cluster) thoth_graphdb_connection_issues{cluster=\"$env\"} and on(cluster) thoth_kafka_connection_issues{cluster=\"$env\"} and on(cluster) thoth_ceph_connection_issues{cluster=\"$env\"}",
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
minor changes needed:

File: grafana-dashboard/overlays/moc-prod/thoth-sli-slo.json (hunk 1 above)
Comment (pacospace) [reply grafana-dashboard/overlays/moc-prod/thoth-sli-slo.json:97]: > lgtm minor changes needed:

It is actually correct, but I suspect there is an issue then: https://github.com/thoth-station/thoth-application/issues/2160

**Review by Gregory-Pereira — APPROVED** [review]
Since the user-api for thoth-frontend-prod has already been addressed here, everything looks good 👍

**Pull request merged**